stripe/src/main/java/com/stripe/android/view/FpxViewModel.kt

```diff
@@ -0,0 +1,34 @@
+package com.stripe.android.view
+
+import android.app.Application
+import android.os.AsyncTask
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.MutableLiveData
+import com.stripe.android.ApiRequest
+import com.stripe.android.PaymentConfiguration
+import com.stripe.android.StripeApiRepository
+import com.stripe.android.StripeRepository
+import com.stripe.android.model.FpxBankStatuses
+
+internal class FpxViewModel(application: Application) : AndroidViewModel(application) {
+    private val context = application.applicationContext
+
+    @JvmSynthetic
+    internal val fpxBankStatuses: MutableLiveData<FpxBankStatuses> = MutableLiveData()
```

Comment:
optional: I've seen `MutableLiveData` be used privately only and exposed publicly as a `LiveData` via a getter (since `LiveData` is a parent of `MutableLiveData`).

This way only the view model can call the mutating methods and the caller can only call the observing methods. Thoughts?

Reply:
Good idea, I like that pattern.